src/scenes/Oracle/index.js

```diff
@@ -157,6 +140,39 @@ class OraclePage extends React.Component {
     );
   }
 
+  isActionButtonDisabled() {
+    const {
+      currentOptionIdx,
+      voteAmount,
+      config,
+      transactions,
+      address,
+    } = this.state;
+
+    // Not within correct times or not the result setter
+    if (config.predictionAction.btnDisabled) {
+      return true;
+    }
+
+    // Already have a pending tx for this Oracle
+    const pendingTxs = _.filter(transactions, { oracleAddress: address, status: TransactionStatus.Pending });
+    if (pendingTxs > 0) {
+      return true;
+    }
+
+    // Did not select an option
+    if (currentOptionIdx === -1) {
+      return true;
+    }
+
+    // Did not enter an amount
+    if ((voteAmount === 0 || Number.isNaN(voteAmount)) && config.predictionAction.showAmountInput) {
+      return true;
+    }
+
```

Comment:
should we also check if the amount input is smaller than my balance?

Reply:
will take care of this when #256 is merged in. don't want to fix it with old logic.